src/olympia/files/tests/test_models.py

```diff
@@ -1409,24 +1412,24 @@ def test_zip_invalid(self):
         obj = self.file_create('search.xml')
         assert obj.get_localepicker() == ''
 
-    @mock.patch('olympia.files.utils.SafeUnzip.extract_path')
-    def test_no_locale_browser(self, extract_path):
-        extract_path.return_value = 'some garbage'
+    @mock.patch('olympia.files.utils.SafeZip.read')
+    def test_no_locale_browser(self, read):
```

Comment:
maybe it's just me but I lack to add a "mock" prefix/suffit to mocked method names. I find it makes it easier to read afterwards.

Reply:
doh, old tests. I'll fix this :)